src/service/api/role/shared/update.patch.ts

```
@@ -0,0 +1,51 @@
+    if (!ScopeManager.isScopeAllowedForSharedSession("<ENTITY>:roles:write", res)) {
+      return;
+    }
+    const id = req.body.target;
+    // Do not allow editing system roles
+    const isSystemRole = Role.isSystemRole(id);
+    if (isSystemRole) {
+      return res.status(statusCodes.clientInputError).json(new ErrorResponse(errorMessages.systemRoleUpdate));
+    }
+    // Scope can be modified only using the access API.
+    if (req.body.scope) {
+      delete req.body.scope;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)